components/ItemCripto/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {Image, Text, View} from 'react-native';
+import {ItemCriptoProps} from '../../utils/interfaces';
+
+import {
+  CriptoContainer,
+  SeconContainer,
+  CriptoName,
+  CriptoPercentage,
+  ImageArrow,
+} from './styles';
+
+const ItemCripto = ({item}: ItemCriptoProps) => (
+  <CriptoContainer>
+    <SeconContainer>
+      <Image source={item.img} />
+      <View>
+        <CriptoName>{item.name}</CriptoName>
+        <Text>{item.symbol}</Text>
+      </View>
+    </SeconContainer>
+    <View>
+      <CriptoName>{item.price}</CriptoName>
+      <CriptoPercentage color={item.percentage > 0 ? true : false}>
```

Comment:
Only use
 <CriptoPercentage color={item.percentage > 0 }>
 
 because item.percentage > 0 is a boolean already

Reply:
Solved in commit: 2c27b58